roles/keystone-setup/tasks/federation.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: set federated domain name
+  ini_file: dest=/etc/keystone/keystone.conf
+            section=federation
+            option=federated_domain_name
+            value=Default
+  when: keystone.federation.sp.oidc.enabled|bool
+  notify: restart keystone services
```

Comment:
This seems somewhat risky, changing a config file in two different places, each with their own restart handler. But it's probably fine until we can get the bootstrap script fixed. Is there anybody working on fixing the bootstrap script?